analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -183,6 +178,23 @@ class Pip(
 
     override fun transformVersion(output: String) = output.removePrefix("pip ").substringBefore(' ')
 
+    private fun runPythonInspector(
```

Comment:
Please add a class that overrides `CommandLineTool` instead, like for example `VirtualEnv` in the same file. This allows it to define supported versions (if required) and will make the tool appear in the output of the `RequirementsCommand`.